src/lib/components/SgtNavigationDrawer/SgtNavigationDrawer.vue

```diff
@@ -80,49 +81,76 @@ import router from "@/router";
 import { SgtNavigationDrawerItem } from "./SgtNavigationDrawerItem.model";
 
 @Component({
-	name: "SgtNavigationDrawer",
-	components: {},
+  name: "SgtNavigationDrawer",
+  components: {},
 })
 export default class SgtNavigationDrawer extends Vue {
-	@PropSync("drawer", { type: Boolean }) syncedName!: boolean;
-	@Prop({ required: true }) navItems: SgtNavigationDrawerItem[];
-	@Prop({ required: false }) brandLogo: string;
-	@Prop({ required: false }) brandName: string;
-	@Prop({ required: false, default: "4rem" }) minVariantWidth: string;
+  @PropSync("drawer", { type: Boolean }) syncedName!: boolean;
+  @Prop({ required: true }) navItems: SgtNavigationDrawerItem[];
+  @Prop({ required: false }) brandLogo: string;
+  @Prop({ required: false }) brandName: string;
+  @Prop({ required: false, default: "60px" }) minVariantWidth: string;
```

Comment:
px won't be responsive, can we have equivalent rem value?

Reply:
This part need not be responsive. It should be 60px in all screen sizes. So, to match with the design I have used 'px' unit.

Reply:
Need to discuss this with Satyawan